reference-data-proxy/src/cronJobs/eStoreCronJobManager.ts

```diff
@@ -2,6 +2,7 @@ import CronJobManager from 'cron-job-manager';
 import { getCollection } from '../database';
 
 const cronJobTimer = new Date();
+
 if (process.env.NODE_ENV === 'dev') {
```

Comment:
I think the value of NODE_ENV is `development` - based on docker-compose files

Reply:
Good spot.